<?xml version="1.0"?>
<launch>
  <arg name="base_frame_id" default="base_link" />
  <arg name="command_timeout_ms" default="1000" />
  <arg name="report_timeout_ms" default="1000" />
  <arg name="loop_rate" default="50.0" />

  <node pkg="pix_rover_driver" name="pix_rover_driver_control_command_node" exec="pix_rover_driver_control_command_node" output="screen">
    <param name="base_frame_id" value="$(var base_frame_id)"/>
    <param name="command_timeout_ms" value="$(var command_timeout_ms)"/>
    <param name="loop_rate" value="$(var loop_rate)"/>
    <remap from="output/can_tx" to="/to_can_bus" />
    <remap from="input/engage" to="/pix_rover_driver/control_command/engage" />
  </node>

  
  <node pkg="pix_rover_driver" name="pix_rover_driver_report_parser_node" exec="pix_rover_driver_report_parser_node" output="screen">
    <param name="base_frame_id" value="$(var base_frame_id)"/>
    <param name="report_timeout_ms" value="$(var report_timeout_ms)"/>
    <param name="loop_rate" value="$(var loop_rate)"/>
    <remap from="input/can_rx" to="/from_can_bus" />
  </node>
  

</launch>
